README.md

```diff
@@ -1 +1,55 @@
-# plugin_sublime
\ No newline at end of file
+![Pieces Sublime Plugin](https://camo.githubusercontent.com/69c990240f877927146712d45be2f690085b9e45b4420736aa373917f8e0b2c8/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7069656365735f7374617469635f7265736f75726365732f7066645f77696b692f5049454345535f4d41494e5f4c4f474f5f57494b492e706e67)
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+
+## Getting Started
+
+Welcome to the Pieces Sublime Plugin.
+The Pieces Sublime plugin is designed to manage your saved materials in Pieces right from within Sublime, and interact seamlessly with Pieces OS.client.
+
+To get started, Check the settings. If you wish to change the llm model you're using or the host, you can do so in the settings `Preferences > Package Settings > Pieces > Settings`.
+
+### Open Saved Material
+
+To open your saved materials, simply open your command palette and run `Pieces: Open Saved Material`. Then, select the asset you wish to open.
+
+#### Edit Saved Materials
+
+To edit a saved material, you'll need to toggle into editor mode. You can do so by tapping `command/ctrl+e`. From there, you will be able to modify the contents of your material and save the changes using `command/ctrl + s`.
+
+#### Copy Saved Material
+
+If you're in an asset sheet, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+
+### Right click to add an asset
+
+To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select `Pieces > Save to Pieces`.
+
+### Copilot Ask
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
+
+### Login/Logout
+
+You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
+
+### Connect with the Pieces Community
+
+#### Join our [Discord Community](https://discord.gg/getpieces)
```

Comment:
Can we link to the Pieces YouTube here and have it be like "View Power Tips & Best Practices"

Reply:
(can we add a section that links to our YouTube)**